walker.py

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""A Random Walk Simulation For Multiple Walkers"""
+
+# Python code for 2D random walk.
+# Source: https://www.geeksforgeeks.org/random-walk-implementation-python/
+import sys
+import numpy as np
+import matplotlib.pyplot as plt
+
+
+def create_walkers(number_of_steps, number_of_walkers):
```

Comment:
Type annotation and return specification might be neat..

Reply:
Good point! thanks